### DISCUSSION TITLE
Can anyone help me understand this error on testConvertSvgToUri (MoodNFT)?

### DISCUSSION THREAD

**Discussion by 0xwtfwtf** [Cyfrin/foundry-full-course-cu#736] [General]
I am trying to test that my function to convert my svg to the image uri is working. I have the same code as Patrick but I am getting a failure on this test....below is a screencap showing the failure with -vvvv. It seems that DeployMoodNft.svgToImageUri is not resulting

Here is the screencap of the error:
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/138697105/204cb320-2d66-467b-aed7-d500fedb383b)

Here is my DeployMoodNft.svgToImageUri function:
```solidity
function svgToImageUri(string memory svg) public pure returns(string memory) {
        string memory baseURL = "data:image/svg+xml;base64,";
        string memory svgBase64Encoded = Base64.encode(bytes(string(abi.encodePacked(svg))));
        return string(abi.encodePacked(baseURL, svgBase64Encoded));
    }

**Comment by theirrationalone** [comment]
There could be typos kind of `uppercase != lowercase` really weird 🙃, You can do `console.log` actualUri -> Copy it -> assign it to expectedUri.

**Reply by DukeLurkin** [reply]
I had the same error and it was the svg code for expectedUri, just a typo.  Must've deleted something when I put it all on the same line.
